src/main/java/hudson/plugins/sonar/SonarPublisher.java

```diff
@@ -441,23 +435,38 @@ public String getDisplayName() {
 
     /**
      * @return all configured {@link hudson.plugins.sonar.SonarInstallation}
+     * @deprecated Global configuration has migrated to {@link SonarGlobalConfiguration}
      */
+    @Deprecated
     public SonarInstallation[] getInstallations() {
       return installations;
     }
 
+    /**
+     * @deprecated Global configuration has migrated to {@link SonarGlobalConfiguration} 
+     */
+    @Deprecated
     public boolean isBuildWrapperEnabled() {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)